Vocab document Symbols in ISO: https://github.com/metanorma/metanorma… (#573)
metanorma/metanorma.org: author/topics/document-format/xrefs.adoc

link:/author/topics/document-format/collections#indirect-xrefs[indirect cross-references between files in the same collection],
to delimit namespaces and containers from anchor IDs [added in https://github.com/metanorma/metanorma-standoc/releases/tag/v1.7.4].

> If an anchor is not assigned to an entity, AsciiDoc by default assigns GUIDs to be

Reviewer: What does GUIDs have to do with AsciiDoc?

> If an anchor is not assigned to an entity, AsciiDoc by default assigns GUIDs to be

Author: ```suggestion
If an anchor is not assigned to an entity, Metanorma by default assigns GUIDs to be
```

its identifier, prefixed with `_`; e.g. `_01234567-89ab-cdef-0123-456789abcdef`.
In order to keep anchors stable within a single version of the document,

> Metanorma replaces such GUIDs with a GUID calculated through an MD5 Hash of

Author: ```suggestion
Metanorma calculates those GUIDs as an MD5 Hash of
```
